variants/particle_xenon/variant.h

```diff
@@ -69,78 +70,71 @@ extern "C"
+static const uint8_t D11 = (11);
+static const uint8_t D12 = (12);
+static const uint8_t D13 = (13);
+static const uint8_t D14 = (14);
+static const uint8_t D15 = (15);
+static const uint8_t D16 = (16);
+static const uint8_t D17 = (17);
+static const uint8_t D18 = (18);
+static const uint8_t D19 = (19);
+static const uint8_t D20 = (20);
 
 // LEDs
 #define LEDS_NUMBER    1
-#define LED_PRIMARY_PIN            _PINNUM(1, 12)
 #define LED_STATE_ON   1
-#define LED_PRIMARY_IDX 0
-#define LED_RGB_RED_IDX 1
-#define LED_RGB_GREEN_IDX 2
-#define LED_RGB_BLUE_IDX 3
-
-#define LED_RGB_RED_PIN            _PINNUM(0, 13)
-#define LED_RGB_GREEN_PIN          _PINNUM(0, 14)
-#define LED_RGB_BLUE_PIN           _PINNUM(0, 15)
+
 #define BOARD_RGB_BRIGHTNESS 0x202020
 
-static const uint8_t LED_BUILTIN   = LED_PRIMARY_PIN;
-static const uint8_t LED_RGB_RED   = LED_RGB_RED_PIN;
-static const uint8_t LED_RGB_GREEN = LED_RGB_GREEN_PIN;
-static const uint8_t LED_RGB_BLUE  = LED_RGB_BLUE_PIN;
+static const uint8_t LED_BUILTIN   = (22);
+static const uint8_t LED_RGB_RED   = (23);
+static const uint8_t LED_RGB_GREEN = (24);
+static const uint8_t LED_RGB_BLUE  = (25);
 
 // Buttons
 #define BUTTONS_NUMBER 2
-#define BUTTON_DFU                  _PINNUM(0, 11)
-#define BUTTON_FRESET               _PINNUM(0, 03) // A0
+#define BUTTON_DFU                  (20)
+#define BUTTON_FRESET               (21) // A0
 #define BUTTON_PULL    NRF_GPIO_PIN_PULLUP
 
-static const uint8_t BUTTON_MODE = PIN_D20;
+static const uint8_t BUTTON_MODE =  (20);
 
 // Antenna
-#define ANTENNA_SWITCH_1            _PINNUM(0, 24)
-#define ANTENNA_SWITCH_2            _PINNUM(0, 25)
+#define ANTENNA_SWITCH_1            (26)
+#define ANTENNA_SWITCH_2            (27)
 
 // NFC
-#define NFC1                        _PINNUM(0, 9)
-#define NFC2                        _PINNUM(0, 10)
+#define NFC1                        (28)
+#define NFC2                        (29)
 
 /*
  * Analog pins
  */
-#define PIN_A0                      _PINNUM(0, 3)
-#define PIN_A1                      _PINNUM(0, 4)
-#define PIN_A2                      _PINNUM(0, 28)
-#define PIN_A3                      _PINNUM(0, 29)
-#define PIN_A4                      _PINNUM(0, 30)
-#define PIN_A5                      _PINNUM(0, 31)
-
-static const uint8_t A0  = PIN_A0 ;
-static const uint8_t A1  = PIN_A1 ;
-static const uint8_t A2  = PIN_A2 ;
-static const uint8_t A3  = PIN_A3 ;
-static const uint8_t A4  = PIN_A4 ;
-static const uint8_t A5  = PIN_A5 ;
+#define PIN_A0                      (30)
+#define PIN_A1                      (31)
+#define PIN_A2                      (32)
+#define PIN_A3                      (33)
+#define PIN_A4                      (34)
+#define PIN_A5                      (35)
+
+static const uint8_t A0 = PIN_A0;
+static const uint8_t A1 = PIN_A1;
+static const uint8_t A2 = PIN_A2;
+static const uint8_t A3 = PIN_A3;
+static const uint8_t A4 = PIN_A4;
+static const uint8_t A5 = PIN_A5;
 #define ADC_RESOLUTION    14
 
 /*
```

Comment:
minor thing, you don't have to implement this, xenon has Serial2 mapping, we could put it as 
PIN_SERIAL2_TX/RX here, Serial2 is supported with latest release.

Reply:
And added :)